### DISCUSSION TITLE
Implementation of the epub:type="noteref"

### DISCUSSION THREAD

**Discussion by jhswift** [edrlab/thorium-reader#1795] [Ideas]
I just learned about this valuable method of using a popup window for footnotes, and that it co-exists with the usual end-of-section footnotes. This is unlike the InDesign export options in the Text panel, where you have to choose between popup notes and end-of-section notes.
I also learned, with thanks, that epub:type="noteref"  is supported by Thorium, Colibrio Vanilla reader and Apple books. 

But the implementation in Colibrio includes a link (button) in the popup to "Go to footnote" which takes you to the end-of-section footnotes.  This is a very useful feature.
  
Can this be implemented also in Thorium?

Many thanks
Jim

**Comment by danielweck** [comment]
Hello, this is an interesting design idea, but note that targeted "footnotes" (i.e. note contents, usually located at the end of the current document, or sometimes in another dedicated document) can sometimes be hidden (CSS `display: none`) if they are contained within `aside` HTML elements. In other words, modal popup footnotes rendering is typically mutually-exclusive with in-line display of notes contents.
All that said, interesting feature, worth investigating. Perhaps hidden note contents could be automatically shown when linked to, but I am concerned about showing/hiding HTML fragments inside documents, as this will inevitably trip-up the pagination state.

**Comment by jhswift** [comment]
Many thanks for the reply, Daniel. I was not aware of the issues of hidden footnotes.
But thanks for looking at this. It was new to me when I saw it in the Colibrio Vanilla Reader.
Jim
________________________________
